lms/static/js/spec/groups/views/cohorts_spec.js

```diff
@@ -200,75 +266,60 @@ define(['backbone', 'jquery', 'js/common_helpers/ajax_helpers', 'js/common_helpe
                     verifyHeader(1, defaultCohortName, 0);
                     expect(cohortsView.$('.cohort-management-nav')).not.toHaveClass('is-disabled');
                     expect(cohortsView.$('.cohort-management-group')).not.toHaveClass('is-hidden');
-                    expect(cohortsView.$('.cohort-management-create-form').length).toBe(0);
+                    expect(getAddModal().find('.cohort-management-create-form').length).toBe(0);
                 });
 
                 it("trims off whitespace before adding a cohort", function() {
                     createCohortsView(this);
                     cohortsView.$('.action-create').click();
-                    cohortsView.$('.cohort-create-name').val('  New Cohort   ');
+                    cohortsView.$('.cohort-name').val('  New Cohort   ');
                     cohortsView.$('.action-save').click();
-                    AjaxHelpers.expectRequest(requests, 'POST', '/mock_service/add', 'name=New+Cohort');
+                    expectCohortAddRequest('New Cohort', null);
                 });
 
                 it("does not allow a blank cohort name to be submitted", function() {
-                    createCohortsView(this, 1);
+                    createCohortsView(this, {selectCohort: 1});
                     cohortsView.$('.action-create').click();
-                    expect(cohortsView.$('.cohort-management-create-form').length).toBe(1);
-                    cohortsView.$('.cohort-create-name').val('');
+                    expect(getAddModal().find('.cohort-management-create-form').length).toBe(1);
+                    cohortsView.$('.cohort-name').val('');
                     expect(cohortsView.$('.cohort-management-nav')).toHaveClass('is-disabled');
-                    cohortsView.$('.action-save').click();
+                    getAddModal().find('.action-save').click();
                     expect(requests.length).toBe(0);
                     verifyMessage('Please enter a name for your new cohort group.', 'error');
```

Comment:
@catong Thanks for the detailed explanation. I'm fine with doing this incrementally as you suggest.  Is this in our own style guide so that other teams also stop using "please"?